.github/workflows/.update-conda-packages.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Detect if we are running inside GitHub Actions
+if [[ -z "${GITHUB_ACTIONS:-}" ]]; then
+  echo "💡 Running outside CI: dry run mode enabled."
+  DRY_RUN=1
+else
+  DRY_RUN=0
+fi
+
+# Detect conda-compatible tool
+TOOL=""
+for cmd in micromamba mamba conda; do
+  if command -v "$cmd" &>/dev/null; then
+    TOOL="$cmd"
+    break
+  fi
+done
+
+if [[ -z "$TOOL" ]]; then
+  echo "❌ No conda-compatible tool (micromamba, mamba, conda) found in PATH."
+  exit 1
+fi
+
+CHANNEL="conda-forge"
+echo "🔍 Using $TOOL to check latest versions from $CHANNEL..."
+
+# Iterate over each requirements.txt file
+for req_file in */requirements.txt; do
+  [ -f "$req_file" ] || continue
+
+  service=$(dirname "$req_file")
+  read -r first_line < "$req_file"
+  pkg=$(echo "$first_line" | cut -d= -f1)
+  old_version=$(echo "$first_line" | cut -d= -f2)
+
+  echo "📦 Checking latest version of $pkg for $service..."
+
+  latest_version=$($TOOL search "$pkg" --channel "$CHANNEL" --json | \
+    jq -r '."result"."pkgs"[0].version')
```

Comment:
important; it sounds this logic doesn't get the proper version. I will explain:
```bash
$ conda search mongodb --channel  conda-forge --json | jq -r '."result"."pkgs"[0].version'
null

$ conda search mongodb --channel conda-forge --json   | jq -r '.[].[] | .version'
6.0.0
6.0.0
6.0.1
6.0.1
6.0.2
6.0.12
6.0.12
6.0.16
6.0.16
6.0.16
```

```suggestion
  latest_version=$($TOOL search "$pkg" --channel "$CHANNEL" --json | \
    jq -r '.[].[] | .version' | sort -rV   | head -n1)
```

Reply:
Neither, it should be: micromamba search -c conda-forge mongodb --json|jq -r ".[\"result\"][\"pkgs\"][0]| .version"

Reply:
it seems the logic has the same issue:
https://github.com/mo-dkrz/freva-service-config/pull/1

Reply:
Haha, bummer. I'll see